gateway/ln-gateway/src/lightning/mod.rs

```diff
@@ -161,6 +163,13 @@ pub trait ILnRpcClient: Debug + Send + Sync {
     /// wallet.
     async fn get_funding_address(&self) -> Result<GetFundingAddressResponse, LightningRpcError>;
 
+    async fn sweep_funds(
```

Comment:
It is kind of confusing to have a `sweep_funds` function that takes an amount. Why not just `withdraw`?

Reply:
Renamed everything to withdraw!